docs/fundamentals/flow/monitoring-flow.md

```diff
@@ -91,6 +91,51 @@ On the other hand, If you want to only enable the monitoring on a given Executor
 Flow().add(...).add(uses=MyExecutor, monitoring=True)
 ```
 
+### Enable monitoring with replicas and shards
+
+When using replicas or shards (see this how-to scale-out) all of the duplicate Executor will expose their own metrics.
```

Comment:
This doesn't look like a proper internal link

Reply:
Oh that's because I forgot to add the link